### DISCUSSION TITLE
Change background color in cosmograph.app?

### DISCUSSION THREAD

**Discussion by ymerouani** [cosmograph-org/cosmograph-issues#5] [Q&A] [answered]
Hello!
Thank you for this incredible tool!

I was wondering if it is possible to somehow change the background color on cosmograph.app somehow? Maybe by editing something through the browser? I cannot find a way to do it. I would love to change the link color from the default one, as well as the background color to make things clearer for an image of my graph to put in a presentation.

**Comment by rokotyan** [comment, marked as answer]
Hi @matriim, unfortunately it's not supported at this moment. Maybe one day we'll add it as a feature but we don't have a timeline for that yet.
